sale_discount_display_amount/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Discount Display Amount',
+    'summary': """
+        This addon intends to display the amount of the discount computed on
+        sale_order_line and sale_order level""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'http://www.acsone.eu',
```

Comment:
URL should be `https://github.com/OCA/sale-workflow`

Reply:
Done!